src/main/resources/application.properties

```diff
@@ -3,3 +3,9 @@ spring.datasource.password=root
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 spring.datasource.url=jdbc:mysql://localhost:3306/hermes
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=hermesconnor1@gmail.com
```

Comment:
datele de logare nu ar trebui sa apare NEVER intr-un repo

poti lasa port-ul, ca ala e mai generic, but never datele de logare

Reply:
@fujinuji got it, o sa le encriptam